### PR TITLE
Move to a 2-branch model

### PR DESCRIPTION
**Pull request by longrunningprocess**
## Description

These changes are in response to a retrospective item about streamlining out change management processes for LF.

### Type of Change

- Engineering

## Screenshots

N/A

## How Has This Been Tested?

This will be tested in practice since it's mostly about our manual processes but changes will still need to be made to the TC deployment job once this gets merged into `develop`.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have reviewed the title/description of this PR which will be used as the squashed PR commit message
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] Update TC deployment job for "staging" once this is merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,16 +16,26 @@ To report an issue using the **Language Forge** service, email "languageforgeiss
...
-
-| Master Branch | Staging Branch (for testing) | Production Branch |
-| ------------- | --------- | ----------- |
-| `master` | `staging` | `prod` |
+We use a modified [Gitflow](http://nvie.com/posts/a-successful-git-branching-model/) process to manage changes.
+
+### Typical ###
+1.  Create a branch off of `develop`
+1.  Create a PR back into `develop`
+1.  Once approved and merged, test those changes in the staging environment
```
Comment (megahirt) [inline README.md:24]:
```suggestion
1.  Once approved and merged, test those changes on [qa.languageforge.org](https://qa.languageforge.org)
```

File: README.md [hunk 2]
```diff
@@ -16,16 +16,26 @@ To report an issue using the **Language Forge** service, email "languageforgeiss
...
+1.  Once you are satisifed with the changes, open another PR back into `master`
+1.  Once approved and merged, coordinate with the team to deliver those changes to the production environment
+1.  Team leads will determine when the right time to cut a release, i.e., tag and deploy, typically this should be within a day or two
+
+### Exception ###
+1.  There may be times when it's necessary to create a branch off of `master`
```
Comment (megahirt) [inline README.md:30]:
```suggestion
1.  There may be times when it's necessary to create a branch off of `master` (e.g. a hotfix for the production environment)
```

File: README.md [hunk 3]
```diff
@@ -16,16 +16,26 @@ To report an issue using the **Language Forge** service, email "languageforgeiss
...
+1.  A PR should be created to `master`
+1.  Once approved and merged, coordinate with the team to deliver those changes to the production environment
+1.  Team leads will determine when the right time to cut a release, i.e., tag and deploy, this would likely be right away under these exceptional circumstances
```
Comment (megahirt) [inline README.md:33]: I don't require a PR to merge develop into master, just as I didn't require a PR to merge staging into master.  There should be no unreviewed code in these merges, and my opinion is that a PR will be unnecessary friction that will slow down deployment, especially give the time zone difference.  @longrunningprocess, @rmunn and myself are authorized to merge into master and deploy as desired.  When merging to master and/or deploying, an announcement should be made on Slack to make all parties aware.

**Review by megahirt — APPROVED** [review]
Looks good, thank you for this!

File: README.md (hunk 3 above)
Comment (longrunningprocess) [reply README.md:33]: ok, let's talk through the handoff between the dev and one of us to get it merged into `master` then, without a PR, will we simply rely on the dev to let us know they've finished testing it?

Comment (longrunningprocess) [reply README.md:33]: I'll make the changes and we can discuss at tonight's stand-up

**Pull request merged**